.ocamlformat-ignore

```diff
@@ -0,0 +1 @@
+src/syntax/unicode.ml
```

Comment:
This should not be necessary. There is already an attribute in the ml file to disable ocamlformat. I see there is a small diff, it should be easy to fix the generator to remove the diff.